pkg/skaffold/build/jib/maven.go

```diff
@@ -85,8 +85,24 @@ func getCommandMaven(ctx context.Context, workspace string, a *latest.JibArtifac
 	return MavenCommand.CreateCommand(ctx, workspace, args)
 }
 
+func getSyncMapCommandMaven(ctx context.Context, workspace string, a *latest.JibArtifact) *exec.Cmd {
+	cmd := MavenCommand.CreateCommand(ctx, workspace, mavenBuildArgs("_skaffold-sync-map", a, true))
```

Comment:
`mavenBuildArgs` append either `package` or `prepare-package`. Does that make sense? And I don't really think `_skaffold-sync-map` will work. It should be `jib:_skaffold-sync-map`?

Also we pass `--quiet` for `_skaffold-files_v2`. But I'm not sure if `--quiet` helps, because I didn't see any Maven output in your short demo.

Reply:
Sorry, forget prepending `jib:_`. I see it's added in `mavenBuildArgs`.